kite-data/kite-data-core/src/test/java/org/kitesdk/data/spi/filesystem/TestAvroWriter.java

```diff
@@ -44,9 +44,9 @@
                 String.valueOf(32 * 1024 * 1024)) // 32 MB
             .schema(schema)
             .format("avro")
-            .build());
+            .build(), TEST_SCHEMA);
```

Comment:
Here, I am not passing the same schema as the dataset one so that I can test the scenario where the dataset is updated with the new schema but data is written with the old schema